docs/graphql/manual/deployment/postgres-requirements.rst

```diff
@@ -86,9 +87,32 @@ Here's a sample SQL block that you can run on your database to create the right
     GRANT ALL ON ALL FUNCTIONS IN SCHEMA public TO hasurauser;
 
     -- Similarly add this for other schemas, if you have any.
-    -- GRANT USAGE ON SCHEMA <schema-name> TO hasurauser;
-    -- GRANT ALL ON ALL TABLES IN SCHEMA <schema-name> TO hasurauser;
-    -- GRANT ALL ON ALL SEQUENCES IN SCHEMA <schema-name> TO hasurauser;
+    GRANT USAGE ON SCHEMA <schema-name> TO hasurauser;
```

Comment:
Copy-pasting the entire block as it is would cause errors, so shouldn't these lines be left commented?